game_patch/rf/os/string.h

```diff
@@ -41,6 +41,16 @@ namespace rf
         String(Pod pod) : m_pod(pod)
         {}
 
+        String(const std::string_view str) :
```

Comment:
This can use data before it's initialize and lead to crashes, suggest replacing the function with this, which is functionally equivalent but doesn't have that issue:

```
String(std::string_view str) {
    m_pod.max_len = static_cast<int>(str.size() + 1u);
    m_pod.buf     = string_alloc(m_pod.max_len);

    std::memcpy(m_pod.buf, str.data(), str.size());
    m_pod.buf[str.size()] = '\0';
}
```